qiita_db/portal.py

```diff
@@ -0,0 +1,434 @@
+# -----------------------------------------------------------------------------
```

Comment:
Is the portals object supposed to be getting transaction-ed here or in a later pull request?